### ISSUE TITLE
Cannot load custom_objects argument for KerasModelArtifact

### ISSUE THREAD

**Issue by ballchuuu** [bentoml/BentoML#1745]
**Describe the bug**
Hi, I have already put the default custom object argument into the KerasModelArtifact but I am still getting the following error:

```ValueError: Unknown loss function: TOP1. Please ensure this object is passed to the `custom_objects` argument. See https://www.tensorflow.org/guide/keras/save_and_serialize#registering_the_custom_object for details.```

Would appreciate any help! Thank you! 

**To Reproduce**
```
def TOP1(y_true, y_pred):
    _y_pred = tf.expand_dims(y_pred, axis = -1)  # (BATCH_SIZE, n_classes) ---> (BATCH_SIZE, n_classes, 1) 
    mat = tf.matmul(tf.expand_dims(tf.ones_like(y_true), -1), tf.expand_dims(y_true, axis = 1)) # (BATCH_SIZE, n_classes, 1) x (BATCH_SIZE, 1, n_classes) --> (BATCH_SIZE, n_classes, n_classes)
    score_diffs = tf.matmul(mat, _y_pred) # (BATCH_SIZE, n_classes, n_classes) x (BATCH_SIZE, n_classes, 1) --> (BATCH_SIZE, n_classes, 1)
    score_diffs = tf.squeeze(score_diffs - _y_pred, -1) # (BATCH_SIZE, n_classes)
    loss_by_sample = tf.reduce_sum(tf.nn.sigmoid(tf.square(y_pred)), axis = -1) + \
                      tf.reduce_sum(tf.sigmoid(-score_diffs), axis = -1) + \
                    -tf.squeeze(tf.squeeze(tf.nn.sigmoid(tf.square(tf.matmul(tf.expand_dims(y_true, 1), _y_pred))), -1), -1)
    return tf.reduce_sum(loss_by_sample)

top3accuracy = lambda y_true, y_pred: tf.keras.metrics.top_k_categorical_accuracy(y_true, y_pred, k = 3)

@bentoml.artifacts([KerasModelArtifact('model', {"TOP1": TOP1, "top3accuracy": top3accuracy})
...

bento_svc.pack("model", {"model": model, "TOP1": TOP1, "top3accuracy": top3accuracy})
```

**Expected behavior**
Run without errors

**Comment by ballchuuu** [comment]
Update: Manage to solved this using the following

@bentoml.artifacts([KerasModelArtifact('model', {"TOP1": TOP1, "top3accuracy": top3accuracy})
...

custom_objects = {"TOP1": TOP1, "top3accuracy": top3accuracy}
bento_svc.pack("model", {"model": model,  "custom_objects": custom_objects})

**Comment by parano** [comment]
Thanks for reporting @ballchuuu - this issue has been fixed here: https://github.com/bentoml/BentoML/commit/d6edc4afdce8ded21aecdd5ede8e86dd5cd1526a but has not yet been released, it will be available in the next release 0.13.1
